compiler/noirc_evaluator/src/ssa/ssa_gen/mod.rs

```diff
@@ -2,6 +2,7 @@ pub(crate) mod context;
 mod program;
 mod value;
 
+use acvm::AcirField;
```

Comment:
```suggestion
```

Reply:
The import is necessary for the `is_zero()` call